testing/gdal/APKBUILD

```diff
@@ -36,26 +38,43 @@ build() {
 		--with-curl=/usr/bin/curl-config
 	make
 
-	cd swig/python
+	cp -R swig/python swig/python2
+	cp -R swig/python swig/python3
+
+	cd swig/python2
 	python2 setup.py build
+
+	cd ../python3
+	python3 setup.py build
 }
 
 package() {
 	make DESTDIR="$pkgdir" install
 	chmod -x "$pkgdir"/usr/include/*.h
 }
 
-py() {
-	pkgdesc="$pkgname (python bindings)"
+py2() {
+	pkgdesc="$pkgname (python2 bindings)"
 
-	cd "$builddir"/swig/python
+	cd "$builddir"/swig/python2
 	python2 setup.py install --prefix=/usr --root="$subpkgdir"
 
 	chmod a+x scripts/*
 	install -d "$subpkgdir"/usr/bin
 	install -m755 scripts/*.py "$subpkgdir"/usr/bin/
 }
```

Comment:
Just drop python2 in favour of python3, we're in the process of removing py2 anyway since it'll be EOL soon.

Reply:
Agreed, just remove all the py2 stuff.